<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>activity_summary</class>
 <widget class="QWidget" name="activity_summary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="activity_summary_layout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="date_activity_type_area">
     <item>
      <widget class="QLabel" name="date_time_label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>[Date]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="activity_type_label">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>[Type]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="InfoTable" name="info_table">
     <property name="minimumSize">
      <size>
       <width>310</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>310</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="activity_text_area">
     <item>
      <layout class="QHBoxLayout" name="activity_name_and_flags">
       <item>
        <widget class="QLabel" name="activity_name_label">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>[Activity Name]</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="flags_label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>[Flags]</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="indent">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="description_label">
       <property name="text">
        <string>[Description]</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" rowspan="2">
    <layout class="QGridLayout" name="map_container"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InfoTable</class>
   <extends>QTableWidget</extends>
   <header>activate/app/tables.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>update_page()</slot>
 </slots>
</ui>
